examples/tests/valid/subdir/util.ts

```diff
@@ -0,0 +1,5 @@
+import type extern from "./util.extern";
+
+export const greet: extern["greet"] = async (name) => {
```

Comment:
Once Yoav merges his PR, Can you add an example like this?

```js
let b = new cloud.Bucket();

class E {
  extern "./e.ts"
  pub static inflight f(b: cloud.Bucket);
}

inflight () => {
  lift(b, ["put"]);
  E.f(b);
};

// or

inflight () => {
  lift { b: ["put"] } {
    E.f(b);
  }
};

```

Curios to see the type annotations...

Reply:
> Curios to see the type annotations...

The type won't be different yet, the .d.ts generation does not check any lifting information. I figured I would tackle that after his lift() pr is merged.